person_capture/gui_app.py

```
@@ -5389,14 +5426,186 @@ def _resolve_ffmpeg_bin(self) -> Optional[str]:
+        try:
+            ensure_dir(os.path.dirname(out_path))
+        except Exception:
+            pass
+        for cmd in self._hdr_tonemap_filter_cmds(ffmpeg_bin, frame_idx, crop_xyxy, tmp):
+            try:
+                if os.path.exists(tmp):
+                    os.remove(tmp)
+            except Exception:
+                pass
+            try:
+                cp = subprocess.run(cmd, text=True, capture_output=True, check=False)
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)